### PR TITLE
task-2: creating custom content assets

### PR DESCRIPTION
**Pull request by Vladzio26**

### CODE REVIEW

File: storefront-reference-architecture-master/metadata/task2.xml [hunk 1]
```diff
@@ -0,0 +1,13205 @@
...
+&lt;p&gt;&lt;b&gt;Analogico:&lt;/b&gt; Un televisore analogico non può trasmettere i programmi in alta definizione. È in grado di trasmettere solo programmi creati con una definizione standard, come quelli della televisione normale, della televisione via cavo o dei canali satellitari, inclusi il digitale terrestre, DirectTV o Dish Network negli Stati Uniti.       &lt;br /&gt;&#13;
+&lt;br /&gt;&#13;
+&lt;b&gt;Digitale:&lt;/b&gt; Per "televisione digitale" si intende la programmazione a definizione standard (SDTV), la programmazione a definizione avanzata (EDTV) o la programmazione ad alta definizione (HDTV).        &lt;br /&gt;&#13;
+&lt;br /&gt;&#13;
+&lt;b&gt;SDTV:&lt;/b&gt; La televisione a definizione standard è una televisione analogica con un sintonizzatore ATSC integrato (vedi sotto) che permette di ricevere la programmazione digitale. Questo tipo di televisore potrà trasmettere la programmazione ad alta definizione ma le immagini non appariranno così nitide come appaiono su un vero televisore HD.        &lt;br /&gt;&#13;
+&lt;br /&gt;&#13;
+&lt;b&gt;EDTV:&lt;/b&gt; Si tratta della televisione a definizione avanzata, un televi
```
Comment (Vladzio26) [inline storefront-reference-architecture-master/metadata/task2.xml:5349]: @oleksandra-davydchuk 
Added custom content asset at BM

File: storefront-reference-architecture-master/cartridges/app_storefront_base/cartridge/templates/default/home/homePage.isml [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
         assets.addJs('/js/productTile.js');
         assets.addCss('/css/homePage.css');
     </isscript>
 
     <!-- +1.888.555.0199 --><!--This phone is a requirement to support existing Gomez monitor of SiteGenesis. Demadware customers can remove this.-->
+    
+    <div class="home-main homepage">
+        <iscontentasset aid="custom-content-asset"/>
+    </div>
+
```
Comment (Vladzio26) [inline storefront-reference-architecture-master/cartridges/app_storefront_base/cartridge/templates/default/home/homePage.isml:14]: @oleksandra-davydchuk 
placed content asset in the homePage.isml

File: storefront-reference-architecture-master/metadata/task2.xml (hunk 1 above)
Comment (Vladzio26) [inline storefront-reference-architecture-master/metadata/task2.xml:5349]: @oleksandra-davydchuk 
Added custom content asset at BM

Comment (Vladzio26) [inline storefront-reference-architecture-master/metadata/task2.xml:5350]: @oleksandra-davydchuk 
Added custom content asset at BM

**Comment by Vladzio26** [comment]
image
https://zzrp-011.sandbox.us01.dx.commercecloud.salesforce.com/s/RefArch/home?lang=en_US

**Comment by oleksandra-davydchuk** [comment]
checked, just move implementation to your cartridge

**Pull request merged**